### PR TITLE
Update CODEOWNERS

### PR DESCRIPTION
**Pull request by KZzizzle**
will this conflict with Nik's ownership of the docs directory?

edit codeowners for Welcome dir

## Checklist

- [ ] I think the documentation is well written and up to date
- [ ] Do NOTE have very large media files (e.g. photos, videos, etc)
- [ ] My spell-checker passes
- [ ] If you write a new section, i take ownership adding my user in .github/CODEOWNERS

### CODE REVIEW

**Comment by pcrespov** [comment]
> will this conflict with Nik's ownership of the docs directory?
Only on that subdirectory, which is fine

File: docs/welcome/what_is_osparc.md [hunk 1]
```diff
@@ -3,7 +3,7 @@
 *osparc* will provide users with an interactive approach to effectively develop, extend, validate, certify, document, store, share, and apply models, explore the impact of stimulation parameters, and create predictive, multi-scale, multi-physics models for a wide range of scenarios.
 One of the core elements of the platform is a set of novel neuro- and physiology-functionalized human and animal anatomical models (called NEUROCOUPLE and NEUROFAUNA, respectively). The models will allow simulation of in vivo fields generated by implanted or external stimulators, elucidation of the resulting neuromodulations, and assessment of the changes induced in organ physiology. A brief introduction to the platform's functionality and use can be found here:
 
-[![webinar1](https://user-images.githubusercontent.com/28002886/86373988-54a49a80-bc84-11ea-9692-da7ee4c02604.PNG ':size=50%')](https://www.youtube.com/watch?v=PNeX2yCkzs4&t)
+[![webinar1](https://user-images.githubusercontent.com/28002886/86373988-54a49a80-bc84-11ea-9692-da7ee4c02604.PNG)](https://www.youtube.com/watch?v=PNeX2yCkzs4&t)
```
Comment (pcrespov) [inline docs/welcome/what_is_osparc.md:6]: Have you considered embedding it? With this code
```
<p align="center">
<iframe width="784" height="441" src="https://www.youtube.com/embed/PNeX2yCkzs4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
</p>
```
Current is on the top and the new code is on the bottom
Screen Shot 2020-07-03 at 13 20 59

Comment (KZzizzle) [reply docs/welcome/what_is_osparc.md:6]: I... would have considered it... if I knew that was an option =D Thanks Pedro, yes that sounds like an even better idea.

**Comment by KZzizzle** [comment]
Now I think it's ready =)

**Pull request merged**